modules/oc-mirror-installing-plugin.adoc

```diff
@@ -56,9 +61,27 @@ $ sudo mv oc-mirror /usr/local/bin/.
 
 .Verification
 
+ifndef::restricted[]
```

Comment:
This is close to what I was talking about, but the conditionals aren't working quite right. Both of these are showing in both the v1 and v2 docs. You'll need to figure out what's going on here

Be sure to compare the v1 vs. v2 docs too to make sure what you expect is showing up in each.